Dockerfile

```diff
@@ -158,119 +203,35 @@ ENV \
-
-# Verify nodejs authenticity
-grep "node-${NODE_VERSION}-${ARCHITECTURE}.tar.gz" "SHASUMS256.txt" | shasum -a 256 -c -
-rm -f "SHASUMS256.txt"
-
-# Install Node
-tar xzf "node-$NODE_VERSION-$ARCHITECTURE.tar.gz" -C /usr/local --strip-components=1 --no-same-owner
-rm "node-$NODE_VERSION-$ARCHITECTURE.tar.gz" "SHASUMS256.txt"
-ln -s "/usr/local/bin/node" "/usr/local/bin/nodejs"
-mkdir -p "/opt/nodejs/lib/node_modules/fibers/.node-gyp" "/root/.node-gyp/${NODE_VERSION} /home/wekan/.config"
-
-# Install node dependencies
-npm install -g npm@${NPM_VERSION} --production
-chown --recursive wekan:wekan /home/wekan/.config
-
-# Install Meteor
-cd /home/wekan
-chown --recursive wekan:wekan /home/wekan
-echo "Starting meteor ${METEOR_RELEASE} installation...   \n"
-gosu wekan:wekan curl https://install.meteor.com/ | /bin/sh
-mv /root/.meteor /home/wekan/
-chown --recursive wekan:wekan /home/wekan/.meteor
-
-sed -i 's/api\.versionsFrom/\/\/api.versionsFrom/' /home/wekan/app/packages/meteor-useraccounts-core/package.js
-cd /home/wekan/.meteor
-gosu wekan:wekan /home/wekan/.meteor/meteor -- help
-
-# Build app (Production)
-cd /home/wekan/app
-mkdir -p /home/wekan/.npm
-chown --recursive wekan:wekan /home/wekan/.npm
-chmod u+w *.json
-gosu wekan:wekan meteor npm install --production
-gosu wekan:wekan /home/wekan/.meteor/meteor build --directory /home/wekan/app_build
-cd /home/wekan/app_build/bundle/programs/server/
-chmod u+w *.json
-gosu wekan:wekan meteor npm install --production
-cd node_modules/fibers
-node build.js
-cd ../..
-# Remove legacy webbroser bundle, so that Wekan works also at Android Firefox, iOS Safari, etc.
-rm -rf /home/wekan/app_build/bundle/programs/web.browser.legacy
-mv /home/wekan/app_build/bundle /build
-
-# Put back the original tar
-mv $(which tar)~ $(which tar)
-
-# Cleanup
-apt-get remove --purge --assume-yes ${BUILD_DEPS}
-npm uninstall -g api2html
-apt-get autoremove --assume-yes
-apt-get clean --assume-yes
-rm -Rf /tmp/*
-rm -Rf /var/lib/apt/lists/*
-rm -Rf /var/cache/apt
-rm -Rf /var/lib/apt/lists
-rm -Rf /home/wekan/app_build
-rm -Rf /home/wekan/app
-rm -Rf /home/wekan/.meteor
-
-mkdir /data
-chown wekan --recursive /data
-EOR
-
+# Install node and application sources from build stage
+RUN --mount=type=bind,from=downloads,source=node.tar.gz,target=node.tar.gz \
+    --mount=type=bind,from=build,source=/build,target=/build \
+    # Install NodeJS
+    tar xzf node.tar.gz --strip-components=1 --keep-old-files --no-same-owner && \
+    # Add the wekan user
+    useradd --uid 999 --user-group --system wekan && \
+    # Copy wekan bundle
+    cp -r /build/bundle /wekan && \
+    # Remove the web.browser.legacy platform
+    rm -r /wekan/programs/web.browser.legacy && \
+    # Change ownership of wekan directory
+    chown wekan:wekan -R /wekan
+
+# Change to the wekan user
 USER wekan
 
+# Set the port to listen on
 ENV PORT=8080
-EXPOSE $PORT
+EXPOSE ${PORT}
+
+# Set the maximum stack size
+ENV STACK_SIZE=65500
```

Comment:
This line is the problem 1.

Reply:
I can make it non-configurable if this is what you want, but if you wanted to eliminate this completely, that can be done just bear in mind that it is now the same behaviour as https://github.com/wekan/wekan/blob/main/Dockerfile#L276